manifest.json

```diff
@@ -5,7 +5,39 @@
     "version": "2.2",
     "content_scripts": [{
         "js": [ "content.js" ],
-        "matches": [ "https://*.medium.com/*"]
+        "matches": [ 
+        "https://*.medium.com/*",
+        "https://towardsdatascience.com/*",
+        "https://hackernoon.com/*",
```

Comment:
why do you need to add hackernoon to the list? It doesnt even block you

Reply:
In reality, I noticed that https://towardsdatascience.com/ was blocking, and I looked in other extensions the domains that medium was using. I just copied that list. I also noticed some weren't blocking, but I decided to add them all. Just a suggestion, I'm new to this world